src/building_blocks/deap_deferred.md

```diff
@@ -0,0 +1,10 @@
+# DEAP with a deferred equality check
+
+
+TLSNotary uses the [DEAP](/protocol/2pc/deap.md) protocol to ensure malicious security during the Encryption and Decryption steps.
+
+When using DEAP in TLSNotary, the `User` plays the role of Alice and has full privacy and the `Notary` plays the role of Bob and reveals all of his private inputs after the TLS session with the server is over. (The Notary's private inputs are his TLS session key shares).
+
+The parties run the `Setup` and `Execution` steps of `DEAP` but for performance reasons they defer the `Equality Check` until after the TLS session with the server is over. The rationale for deferring the `Equality Check` is as follows:
```

Comment:
It is not clear to me here, what the rationale for deferring the equality check is. Is it that the user already has an authentic output without the equality check?

Reply:
sry, I was confused, it is not for performance reasons but
Since during the `Equality Check` all of the Notary's secrets are revealed to User, it must be deferred until after the TLS session with the server is over, otherwise the User would learn the full TLS session keys and be able to forge the TLS transcript.